docs/tutorial.rst

```diff
@@ -102,11 +104,415 @@ mlf-core is now configured and ready to be used!
 
 For more details including security precautions please visit :ref:`config` and :ref:`github_support`.
 
-
-.. _Github: https://github.com
-
 Creating a mlf-core project
 ------------------------------
 
+mlf-core offers templates for several machine learning libraries. To get an overview of all available machine learning templates run:
+
+.. code-block:: console
+
+    $ mlf-core list
+
+If you want a more detailed overview you can also run:
+
+.. code-block:: console
+
+    $ mlf-core info <template-handle/type/library>
+
+A more detailed overview on all available templates is provided `here <https://mlf-core.readthedocs.io/en/latest/available_templates/available_templates.html>`_.
+In the follow sections we will create and focus on a Pytorch based template identified under the template handle ``mlflow-pytorch``.
+The outlined processes work the same for all other templates.
+
+To create a mlf-core project run:
+
+.. code-block:: console
+
+    $ mlf-core create
```

Comment:
```suggestion
    $ mlf-core create <<project_path>>
```

Reply:
I want to keep it simple here and therefore won't add any path parameters.